### DISCUSSION TITLE
CS & JS not found

### DISCUSSION THREAD

**Discussion by vinay142** [orchidsoftware/platform#2377] [General]
![Screenshot 2022-09-20 at 9 08 32 AM](https://user-images.githubusercontent.com/42082987/191162147-bd48f4c9-e2a0-464b-8774-f92b5aa10fe0.png)
vendor/orchid/js/manifest.js?id=fa18bd350409678078668239241217f5 net::ERR_ABORTED 404 (Not Found)

**Comment by c-gross** [comment]
Did you run `php artisan orchid:publish` to copy the css/js files in the public folder?

**Reply by vinay142** [reply]
Yes

**Reply by c-gross** [reply]
The files are inside the folder? (public/vendor/orchid/css and /js)
Otherwise the folder permissions are maybe wrong.

**Comment by vinay142** [comment]
No
sudo chown -R $USER:www-data .
sudo find . -type f -exec chmod 664 {} \;
sudo find . -type d -exec chmod 755 {} \;

**Comment by kamuitubeda** [comment]
the solution to this problem is adding: 

'mix_url' => env('ASSET_URL', null),

to config/app.php

full explanation can be found here: [https://github.com/orchidsoftware/platform/issues/2237]
